react/components/CreditCard.tsx

```diff
@@ -63,14 +63,16 @@ interface PaymentSystem {
 }
 
 const IFRAME_APP_VERSION = '0.4.1'
-const LOCAL_IFRAME_DEVELOPMENT = false
+const PORT = 3000
 
-let iframeURL = `https://io.vtexpayments.com.br/card-form-ui/${IFRAME_APP_VERSION}/index.html`
+const iframeURLProd = `https://io.vtexpayments.com.br/card-form-ui/${IFRAME_APP_VERSION}/index.html`
+const iframeURLDev = `https://checkoutio.vtexlocal.com.br:${PORT}/`
 
-if (LOCAL_IFRAME_DEVELOPMENT) {
-  const PORT = 3000
-  iframeURL = `https://checkoutio.vtexlocal.com.br:${PORT}/`
-}
+const { production, query } = __RUNTIME__
+
+const LOCAL_IFRAME_DEVELOPMENT = !production && query.__localCardUi === 'true'
```

Comment:
better to set the local development when the querystring _exists_.

```suggestion
const LOCAL_IFRAME_DEVELOPMENT = !production && query.__localCardUi !== undefined
```